.github/workflows/tests_lgpu_python.yml

```
@@ -253,4 +252,4 @@ jobs:
         with:
           fail_ci_if_error: true
           verbose: true
-          token: ${{ secrets.CODECOV_TOKEN }}
+          token: ${{ secrets.CODECOV_TOKEN }}
```

To fix the rendering issue,
```suggestion
          token: ${{ secrets.CODECOV_TOKEN }}

```